recipes/vmtk/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "vmtk" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/vmtk/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ec52357bf6dfd975eaae592b6b70db67c7767a01278386373588a29cf52997bb
+
+build:
+  number: 0
+  skip:
+    - True  # [win and py27]
+    - True  # [py37]
```

Comment:
I think this is better as a single line, `[(win and py27) or py37]`

Reply:
I agree the two lines have now been merged.

Reply:
@scopatz, when I changed this into one line it tries to build for py37 as well, [which gives](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=12450&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&taskId=bef775d0-a1a2-55a3-9721-17147029baf0&lineStart=340&lineEnd=343&colStart=1&colEnd=78):

```
conda.exceptions.UnsatisfiableError: The following specifications were found to be in conflict:
  - python=3.7
  - vtk[version='<=8.1.0'] -> python[version='>=3.6,<3.7.0a0']

Use "conda search <package> --info" to see the dependencies for each package.
```

Would be OK to change it back into two lines? Or is there something else that is wrong?

Reply:
yeah, you can change it back